src/bun.js/javascript.zig

```diff
@@ -2997,8 +2998,9 @@ pub const VirtualMachine = struct {
 
         for (this.preload) |preload| {
             var result = switch (this.transpiler.resolver.resolveAndAutoInstall(
-                this.transpiler.fs.top_level_dir,
-                normalizeSource(preload),
+                // FIXME: should be defaulting to cwd
+                preload.root_dir orelse this.transpiler.fs.top_level_dir,
```

Comment:
i think this.transpiler.fs.top_level_dir is the cwd?

Reply:
I believe it's the location of package.json